### DISCUSSION TITLE
무엇이 잘못되었는지 모르겠습니다.

### DISCUSSION THREAD

**Discussion by peterkangdev** [naverz/zepeto-studio-kor#308] [5. Q&A]
다음 코드에서 제가 무엇을 잘못했는지 아시는 분 있으세요?

<img width="925" alt="스크린샷 2022-02-03 오후 1 53 32" src="https://user-images.githubusercontent.com/30133892/152283456-a87f9ecc-5b30-4417-9697-cd6280cf5201.png">

트리거가 실행되면 다음과 같은 에러가 나옵니다.

<img width="457" alt="스크린샷 2022-02-03 오후 1 56 46" src="https://user-images.githubusercontent.com/30133892/152283692-15a7a454-d700-4b3d-9143-03169e4f18af.png">

당연히 OnGetItemCube 가 있고요..

<img width="677" alt="스크린샷 2022-02-03 오후 1 55 49" src="https://user-images.githubusercontent.com/30133892/152283730-6e8de42b-7a07-411d-adba-765951f45879.png">

문제가 있나요??
왜 안되는 걸까요??

**Comment by mayko1234** [comment]
<del>안녕하세요, 제 경험으론 함수가 비어있을때 저런 에러가 나더라구요.
OnGetItemCube()에 console.log("test"); 이런 라인을 넣으신다음 플레이해보세요. </del>

**Reply by peterkangdev** [reply]
로그를 넣어도 똑같습니다.

**Reply by mayko1234** [reply]
아 혹시, 저 FirstItems 스크립트가 오브젝트에 추가되어있나요? 
![image](https://user-images.githubusercontent.com/97988822/152423354-b07456b2-9c28-4a53-8f93-24270eb934a5.png)
제가 방금 똑같은 에러를 만들었는데, 오브젝트에서 FirstItems 스크립트를 아예 빼버리고, ItemCube클래스에서 참조를 하니까 똑같은 에러가 발생하더라구요.
